dmscripts/notify_buyers_to_award_closed_briefs.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, date, timedelta
+
+import dmapiclient
+from dmutils.email.dm_notify import DMNotifyClient
+from dmutils.email.exceptions import EmailError
+from dmutils.formats import DATE_FORMAT
+
+from dmscripts.helpers import logging_helpers, date_helpers, brief_data_helpers
+from dmscripts.helpers.logging_helpers import logging
+
+logger = logging_helpers.configure_logger({'dmapiclient': logging.INFO})
+
+
+def _filter_brief_users_by_optional_user_id_list(brief, user_id_list):
+    """Filter the brief's users by restricted user ID list if present
+    """
+    if user_id_list:
+        return filter(lambda user: user['id'] in user_id_list, brief['users'])
+    else:
+        return brief['users']
+
+
+def _filter_briefs_by_optional_user_id_list(closed_briefs, user_id_list):
```

Comment:
I'm not sure this function name reflects what it does. It filters them by if they have users, and optionally by the user id list?

Reply:
I'll rename and add a docstring - something like `Return a list of briefs if they have at least 1 user in the given ID list`?